base/applications/network/telnet/src/tnmain.cpp

```diff
@@ -146,14 +146,18 @@ struct cmdHistory * cfgets (char *buf, unsigned int length, struct cmdHistory *c
 						if (InputRecord.Event.KeyEvent.dwControlKeyState &
 							(LEFT_CTRL_PRESSED | RIGHT_CTRL_PRESSED)) {
 							unsigned int j, k;
-							for (j = cursor; j <= current; j++)
-								if (buf[j+1] == ' ' || (j+1)==current)
-									break;
-								for (k = ++j; k <= current; k++)
-									if (buf[k] != ' ' || k == current) {
-										cursor = k == current ? --k : k;
-										break;
-									}
+                            for (j = cursor; j <= current; j++)
+                            {
+                                if (buf[j + 1] == ' ' || (j + 1) == current)
+                                    break;
+                                for (k = ++j; k <= current; k++)
+                                {
+                                    if (buf[k] != ' ' || k == current) {
+                                        cursor = k == current ? --k : k;
+                                        break;
+                                    }
+                                }
+                            }
```

Comment:
Are there real changes here? Or is it just coding style changes?
This is hard to review.
If that's just coding style, I suggest to revert.